### ISSUE TITLE
Welder_act no longer works on walls of any type

### ISSUE THREAD

**Issue by Coolrune206** [ParadiseSS13/Paradise#22109]
**Issue Description**:

Welders no longer work on any walls.

**What did you expect to happen**: 

Welder + Harm Intent should weld down a regular wall.
After other steps, applying welder should proceed with deconstruction of R-Wall.

**What happened instead**:

Welder + Harm Intent did nothing to regular wall.
Other steps with other tools worked on R-Wall until welder stage reached, then it did not work.

**Why is this bad/What are the consequences**:

THE DANG WALLS WON'T DIE!

**Steps to reproduce the problem**:

Apply welder to wall.

**When did the problem start happening**:

Basically just now. Likely related to https://github.com/ParadiseSS13/Paradise/pull/21950.

**Extra information**:

I was going to say this is a truly cutting issue but Kyet just stole that pun from me :(

**Comment by Contrabang** [comment]
Fixed
